Themes/default/css/index.css

```diff
@@ -2794,7 +2798,7 @@ dl.addrules dt.floatleft {
 	background: #f0f4f7;
 }
 .windowbg:nth-of-type(odd) {
-	background: #fdfdfd; /* needs new color */
+	background: #EEEEEE;
```

Comment:
Wasn't the old one fine?

On Fri, Feb 6, 2015 at 7:57 AM, Mert Alınbay <notifications@github.com>
wrote:

> In Themes/default/css/index.css
> <https://github.com/SimpleMachines/SMF2.1/pull/2717#discussion_r24244903>:
>
> > @@ -2794,7 +2798,7 @@ dl.addrules dt.floatleft {
> >  	background: #f0f4f7;
> >  }
> >  .windowbg:nth-of-type(odd) {
> > -	background: #fdfdfd; /* needs new color */
> > +	background: #EEEEEE;
>
> I really really bad at finding good color...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SimpleMachines/SMF2.1/pull/2717/files#r24244903>.
>